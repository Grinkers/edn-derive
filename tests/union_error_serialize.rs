use edn_derive::Serialize;

#[derive(Serialize)]
union BadType {
    value: usize,
}

fn main() {}
