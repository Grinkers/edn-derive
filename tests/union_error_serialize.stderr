error: `union` types aren't supported
 --> $DIR/union_error_serialize.rs:4:1
  |
4 | union BadType {
  | ^^^^^
